build/lib/release.sh

```diff
@@ -284,7 +284,7 @@ function kube::release::create_docker_images_for_server() {
     local images_dir="${RELEASE_IMAGES}/${arch}"
     mkdir -p "${images_dir}"
 
-    local docker_registry="${KUBE_DOCKER_REGISTRY:-gcr.io/google-containers}"
```

Comment:
https://github.com/kubernetes/kubernetes/pull/49066 should have fixed the `anago` case. let's see if all of the jobs pass now...